actionpack/lib/action_controller/metal/rendering.rb

```diff
@@ -160,6 +160,12 @@ def inherited(klass)
     #         render "posts/new", status: :unprocessable_entity
     #         # => renders app/views/posts/new.html.erb with HTTP status code 422
     #
+    # `:variants`
+    # :  This tells Rails to look for template variations of the same format.
+    #
+    #         render variants: [:mobile]
+    #         # => renders app/views/home/index.html+mobile.erb
```

Comment:
To keep inline with some of the earlier examples, and make the template name a bit clearer, how about passing the template path?
```suggestion
    #         render "posts/index", variants: [:mobile]
    #         # => renders app/views/posts/index.html+mobile.erb
```

Reply:
Addressed here: fb446296ba958fdb767196cdaad0e11ad2433a06